e2e-tests/cypress/support/index.d.ts

```diff
@@ -0,0 +1,250 @@
+declare namespace Cypress {
+  interface Chainable {
+
+    /**
+     * Custom command to add a note to a signal.
+     * @example cy.addnote('This is a note');
+    */
+    addNote: (noteText: string) => Chainable<Element>;
```

Comment:
Are these really chainable? That would mean you could do `cy.addNote('Foo').addNote('Bar')`.

Reply:
What do you think about doing these declarations in the files themselves? 

Then you could do the following:

```   
   /**
     * Custom command to add a note to a signal.
     * @example cy.addnote('This is a note');
    */
    addNote: typeof addNote
```

And then import that file in this index.d.ts file. The benefit is having both the type, implementation and documentation colocated.